.cspell.json

```
@@ -282,7 +282,9 @@
     "appinfo",
     "Applanix",
     "apriltag",
+    "apriltags",
     "arange",
+    "arccos",
```

How about also adding related words?

```suggestion
    "arccos",
    "arcsin",
    "arctan",
```

---

Oh, `arctan` was already there.